src/lib/progress-bar/progress-bar.scss

```diff
@@ -120,12 +112,26 @@ $mat-progress-bar-piece-animation-duration: 250ms !default;
   &[mode='buffer'] {
     .mat-progress-bar-background {
       // Avoids stacked animation tearing in Firefox >= 57.
-      @include _noop-animation();
```

Comment:
For keeping the files clean: Since the `noop-animation` mixin is no longer used, we should be able to remove the SaSS import above.